apps/etl-func/avro/message-status.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "messageStatus",
+  "type": "record",
+  "namespace": "dto",
+  "doc": "Kafka JS schema for cosmos api container 'message-status'",
+  "fields": [
+    {
+      "name": "schema_version",
+      "doc": "An integer that indicates the version of the schema.",
+      "type": "int"
+    },
+    {
+      "name": "op",
+      "type": {
+        "type": "enum",
+        "doc": "List of operations that can be done to a message.",
+        "name": "MessageStatusCrudOperation",
+        "symbols": ["CREATE", "UPDATE"]
+      }
+    },
+    {
+      "name": "id",
+      "doc": "An id that identifies the status of the message inside the message-status collection.",
+      "type": "string"
+    },
+    {
+      "name": "message_id",
+      "doc": "An ULID that identifies the message inside the message collection.",
+      "type": "string"
+    },
+    {
+      "name": "status_version",
+      "doc": "An integer that indicates the version of the status.",
+      "type": "int"
+    },
```

Comment:
what is the purpose of this field? it points to the latest status available?

Reply:
message-status is a versioned collectionn, so this value indicates the current version of the document

Reply:
and why it is needed? should we push a new updated `message` to the queue for every status update? 🤯

Reply:
Aren't we planning to use cosmos change feed on message-status collection? The function would be triggered for every status update

Reply:
Ok, but I was expecting that for every change in `message-status`, we would enqueue a new `message-status` event and not a `message` one.

Reply:
in fact this is the message-status avro